pesudo-code/deniz.md/deniz-pesude-code.md

```diff
@@ -0,0 +1,73 @@
+Technologic
```

Comment:
@udenizdemirbilek why is this file inside another folder

Reply:
Deleted sub-folder "deniz" under "pseudo-code".
Submission should appear under "pseudo-code", could you check please?